tsdb/db_test.go

```diff
@@ -3999,11 +4016,13 @@ func TestQueryHistogramFromBlocks(t *testing.T) {
 		)
 	})
 
-	t.Run("overlapping blocks with only histograms", func(t *testing.T) {
+	t.Run("serial blocks with only histograms and only float in a series", func(t *testing.T) {
```

Comment:
The description confuses me. Maybe "serial blocks: histograms, floats, histograms" or similar?

Reply:
I meant that a block has either histograms or float and not both